core/migrations/0055_update_templates_with_cyverse.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+def try_update_iplant_to_cyverse(apps, schema_editor):
+    try:
+        EmailTemplate = apps.get_model("core", "EmailTemplate")
+        template = EmailTemplate.objects.get(email_address='support@iplantcollaborative.org')
+        template.email_address = u'support@cyverse.org'
+        template.email_footer = u'CyVerse Atmosphere Team'
+        template.save()
+    except:
+        pass
```

Comment:
I tested this with bogus values that would throw the exception. This is safe to run on jetstream.